### DISCUSSION TITLE
想要使用fasterdeployserver部署pp-ocr-v3模型和pp-structure模型输出结构化的数据应该怎么做？

### DISCUSSION THREAD

**Discussion by carbmax** [PaddlePaddle/PaddleOCR#12354] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：docker GPU fastdeploy-2.15.0

目前使用fasterdeployserver 形式部署pp-ocr-v3可以输出非结构化的文字，想要结合PP-Structure输出结构化的数据。怎么使用fastdeployserver结合两个模型？

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): no

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by tink2123** [comment]
可以参考下PP-ChatOCRv2的部署：https://aistudio.baidu.com/application/detail/10368
